sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobOutputStream.java

```diff
@@ -34,33 +34,33 @@
     /**
      * Holds the {@link BlobAccessConditions} object that represents the access conditions for the blob.
      */
-    private BlobAccessConditions accessCondition;
+    private final BlobAccessConditions accessCondition;
 
     private AppendPositionAccessConditions appendPositionAccessConditions;
 
     /**
      * Used for block blobs, holds the block id prefix.
      */
-    private String blockIdPrefix;
+    private final String blockIdPrefix;
 
     /**
      * Used for block blobs, holds the block list.
      */
-    private TreeMap<Long, String> blockList;
+    private final TreeMap<Long, String> blockList;
```

Comment:
It's called blockList given that it is used by BlockBlob's Put Blob List method. Given that this is the property on the class this can be changed to Map.

Reply:
How do we like to handle this?  Do you want me to update the name as part of this same PR?  or should we create a new issue to request this change to have an independent commit for that refactoring?

Reply:
Issue created. Since there is more things to address than just names or type updating